Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "cpython"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
Is there a reason for this version bump?  We usually do that in a separate commit when we make the release (e.g. c7224ad2d 2e1ac24d6 b7090799b6)